cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh

```diff
@@ -44,13 +43,22 @@ model_version=$(az ml model show --name $model_name --label $model_label --regis
 
 # 3. Deploy the model to an endpoint
 # Create online endpoint 
-az ml online-endpoint create --name $endpoint_name $workspace_info  || {
+az ml online-endpoint create --name $endpoint_name $workspace_info || {
     echo "endpoint create failed"; exit 1;
 }
 
+# 3.1 Setup Deployment Parameters
+max_concurrent_request=2  # the maximum number of concurrent requests supported by the endpoint
+
+# Note: We have set the value of `max_concurrent_request` to 2, 
+# as we are utilizing the `Standard_NC6s_v3` SKU for deployment, which has one GPU. 
+# If you are using a larger SKU, please increase this value to get the maximum performance.
+
 # Deploy model from registry to endpoint in workspace
 az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
```

Comment:
@gauravrajguru - let's separate the traffic setting

Reply:
Will do it in later PR of extra parameters. Thanks,